gateway-service/src/main/java/org/zowe/apiml/gateway/security/mapping/model/MapperResponse.java

```diff
@@ -28,5 +35,40 @@ public class MapperResponse {
     @JsonProperty("racfReturnCode")
     private int racfRc;
     @JsonProperty("racfReasonCode")
-    private int reasonCode;
+    private int racfRs;
+
+    @InjectApimlLogger
+    private final ApimlLogger apimlLog = ApimlLogger.empty();
+
+    public String toString() {
```

Comment:
Why we need a special toString() method if the class is annotated as @Data?

Reply:
Could be deleted.

Reply:
toString() method removes unnecesary string representation of apimlLogger.